recipes/minio-client/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set goorg = "github.com/minio" %}
+{% set goname = goorg + "/mc" %}
+{% set version = "2022-04-16T21-11-21Z" %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
+
+package:
+  name: minio-client
+  version: {{ version.replace("-", ".").replace("T", ".").replace("Z", "") }}
+
+source:
+  - folder: {{ pkg_src }}
+    url: https://{{ goname }}/archive/refs/tags/RELEASE.{{ version }}.tar.gz
+    sha256: 94c1d0b2450b8024f70142ac15a1cddfe007779de9f52264d7e572f4ab6de801
+
+build:
+  skip: True  # [win]
+  number: 0
+  script:
+    - pushd {{ pkg_src }}
+    - go install -v -ldflags "-X main.revision=conda-forge" .
+    - go-licenses save --ignore {{ goname }} --ignore {{ goorg }}/pkg --save_path={{ SRC_DIR }}/go-licenses .
```

Comment:
If that works, would be interesting for https://github.com/conda-forge/conda-forge.github.io/issues/1052

I tried it already in https://github.com/conda-forge/go-sops-feedstock/pull/9 but no bueno so far and was on holiday so didn't have time.

Reply:
yep, we'll see if it works!